spec/models/landing_page_spec.rb

```diff
@@ -1,103 +1,64 @@
 RSpec.describe LandingPage do
-  let(:content_item) do
-    http_response = instance_double(RestClient::Response)
-    allow(http_response).to receive(:body).and_return(
-      {
-        "base_path" => "/landing-page",
-        "title" => "Landing Page",
-        "description" => "A landing page example",
-        "locale" => "en",
-        "document_type" => "landing_page",
-        "schema_name" => "landing_page",
-        "publishing_app" => "whitehall",
-        "rendering_app" => "frontend",
-        "update_type" => "major",
-        "details" => {},
-        "routes" => [
-          {
-            "type" => "exact",
-            "path" => "/landing-page",
-          },
-        ],
-      }.to_json,
-    )
-    GdsApi::Response.new(http_response)
-  end
+  subject(:landing_page) { ContentItemFactory.build(landing_page_example) }
```

Comment:
Why is `ContentItemFactory.build` being called here rather than `described_class.new`?

Reply:
Oh yeah, this is just a failure to follow all the way down (I was fixated on removing the helper, forgot this example could go even further). Thanks! fixed now.